src/types.rs

```diff
@@ -343,24 +343,50 @@ pub struct OrderState {
     pub finalize: String,
     /// The certificate URL, which becomes available after finalization
     pub certificate: Option<String>,
+    /// The certificate that this order is replacing, if any
+    #[serde(deserialize_with = "deserialize_static_certificate_identifier")]
+    #[serde(default)]
+    pub replaces: Option<CertificateIdentifier<'static>>,
 }
 
 /// Input data for [Order](crate::Order) creation
 ///
 /// To be passed into [Account::new_order()](crate::Account::new_order()).
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
-pub struct NewOrder<'a> {
+pub struct NewOrder<'a, 'b> {
+    /// The [`CertificateIdentifier`] of a previously issued certificate being replaced by the order
+    pub(crate) replaces: Option<CertificateIdentifier<'b>>,
     /// Identifiers to be included in the order
     identifiers: &'a [Identifier],
 }
 
-impl<'a> NewOrder<'a> {
+impl<'a, 'b> NewOrder<'a, 'b> {
```

Comment:
Do we actually need a second lifetime for this? I'd expect we can just reuse `'a` for this.

Reply:
Switched to one lifetime.